README.md

```diff
@@ -113,3 +113,30 @@ We are passionate about improving this project, and glad to accept help to make
 
 **Feature Requests:** If you have a need that is not met by the current implementation, please let us know (via a new issue).
 This feedback is crucial for us to deliver a useful product. Do not assume that we have already thought of everything, because we assure you that is not the case.
+
+## Naming convention
+
+By adding support for a new resource, 3 files are required: self-contained module, test and example.
+The following is a summary of the code structure and naming conventions for the oneview-ansible modules.
+
+**Modules**
+
+The modules are located in **library** folder, all modules need to be self-contained, 
```

Comment:
Just a small suggestion:

Modules are located in **library** folder. All modules need to be self-contained, 
without external dependencies except hpOneView.